ttter/accounts/urls.py

```diff
@@ -0,0 +1,11 @@
+
+from django.urls import path, include
+from . import views
+
+app_name = 'accounts'
+
+urlpatterns = [
+    path('create_data_input/', views.UserDataInput.as_view(), name="create_data_input"),
+    path('create_confirm/', views.UserDataConfirm.as_view(), name="create_confirm"),
+    path('create_user/', views.UserDataCreate.as_view(), name='create_user'),
```

Comment:
```suggestion
    path('user_data_input/', views.UserDataInput.as_view(), name="user_data_input"),
    path('user_data_confirm/', views.UserDataConfirm.as_view(), name="user_data_confirm"),
    path('user_data_create/', views.UserDataCreate.as_view(), name='user_data_create'),
```
このようにview名と一致させておいた方がわかりやすいですね。とはいえ直すかどうかはお任せします。